src/types/slash.ts

```diff
@@ -13,13 +18,33 @@ export interface InteractionApplicationCommandOption {
   options?: InteractionApplicationCommandOption[]
 }
 
+export interface InteractionChannelPayload {
+  id: string
+  name: string
+  permissions: string
+  type: ChannelTypes
+}
+
+export interface Dict<T> {

```

Comment:
I think moving this to another file will be better since it might be useful in other things too

Reply:
Done.